### PR TITLE
Adds Arts/Sports/Money lore to Skrell

### PR DESCRIPTION
**Pull request by ParadoxonKomplikon**
What is says on the tin. Moved the Castes section up for better reading.

### CODE REVIEW

File: Species/The Skrell.md [hunk 1]
```diff
@@ -161,6 +183,20 @@ Teachers and Professors have the same purpose, having the ability to vote the la
...
+
+This means Skrell cannot hoard money, and are instead incentivized to spend it, be it by buying goods and services, or investing it in various endeavors. If a Skrell wishes to preserve money for future projects, or simply for financial safety, their only recourse is to loan that money, without interest, to the City-State, for a set duration decided at the start of the loan. When the government gives back the money, the bills are freshly printed for a new 2-year long period of use. 
+
+That way, the Skrellian City-States are consistently supplied with their citizens’ money, negating entirely the need for taxes or banks in their society. Since money is constantly printed and destroyed, inflation is a very minor phenomenon, and the gap of wealth between the various casts, while still existent, is far less consequent than in humans societies.
```
Comment (Raptor1628) [inline Species/The Skrell.md]: Did you mean far less consequential?

File: Species/The Skrell.md [hunk 2]
```diff
@@ -161,6 +183,20 @@ Teachers and Professors have the same purpose, having the ability to vote the la
...
+
+However, the human apparition on the galactic market has brought imbalance to the Skrellian system.  Humans have no use in the perishable money of the Skrell, forcing the Skrell to use human money for inter-species exchanges. Luckily, the skrellian technological advantage allowed them to sell a lot of goods to mankind, injecting a lot of foreign currencies in the skrellian market. 
+
+While human money is technically unusable in the Skrellian territories, most City-States accept it in exchange of freshly printed Skrellian bills, which the government then uses for exterior trade, and under strict conditions, to exchange with Skrellian corporations wishing to trade with humans. Nonetheless, a noticeable proportion of Skrell have found another use to human currencies: Non-perishable, it allows them to keep money for as long as they wish, only exchanging it against Skrellian currency when the need arises.
```
Comment (Raptor1628) [inline Species/The Skrell.md]: found another _advantage of_ human currency: _non-perishability_

File: Species/The Skrell.md (hunk 1 above)
Comment (ParadoxonKomplikon) [reply Species/The Skrell.md]: Probably, yeah. Call it a translation error, as "consequent" means "consequential" in French. Fixing it.

File: Species/The Skrell.md (hunk 2 above)
Comment (ParadoxonKomplikon) [reply Species/The Skrell.md]: Eh, fair enough. Fixing it !

**Comment by Raptor1628** [comment]
Looks great man, was a pleasure to read.

**Pull request merged**